commands/geosearch.md

```diff
@@ -36,6 +39,10 @@ When additional information is returned as an array of arrays for each item, the
 2. The geohash integer.
 3. The coordinates as a two items x,y array (longitude,latitude).
 
+@history
+
+* `>= 6.2`: COUNT can take negative number to return some matches ASAP.
```

Comment:
```suggestion
* `>= 6.2`: COUNT accepts a negative value and returns as soon as possible.
```

Reply:
actually GEOSARCH is a new command, we need to drop this part